pimd/pim_cmd.c

```diff
@@ -6861,6 +6877,34 @@ DEFUN (no_ip_pim_packets,
 	return CMD_SUCCESS;
 }
 
+DEFPY (igmp_group_watermark,
+       igmp_group_watermark_cmd,
+       "ip igmp watermark-warn (10-60000)$limit",
```

Comment:
are you planning to make this not only "warn" in the future? if not, then the naming isn't right, IMO.

Reply:
I'm regarding `igmp_watermark_limit` which is not consistent with the CLI name

Reply:
@ton31337 Thanks for reviewing.
This is going to be warning always.
This variable stores the limit and command is to generate warning. That's why named as such.
Please suggest if you have a better naming in mind. Shall change it accordingly.

Reply:
All good, I just hooked on `watermark-warn` and `igmp_watermark_limit` variable.